designer/client/src/components/graph/CursorMask.ts

```diff
@@ -1,31 +1,18 @@
-import { css } from "@emotion/css";
 import { CSSProperties } from "react";
 
 export class CursorMask {
-    private static instance: CursorMask;
-    private readonly cursorMask: HTMLElement;
-
-    constructor() {
-        this.cursorMask = document.createElement("DIV");
-        this.cursorMask.className = css({
-            cursor: "not-allowed",
```

Comment:
@JulianWielga Do you remember what's the reason for this initial cursor state?

Reply:
this element is appended later, so there is no real reason. it will simply signal the default value and what it is for.